lib/engine/game/g_18_usa/step/p11_track.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Engine
+  module Game
+    module G18USA
+      module P11Track
```

Comment:
Does this combine nicely with P7(Track Engineers) and/or P17 (Great Northern Railway)?

Reply:
Good idea to test this. I made a small update to allow P7 and P11 to coexist. P17 is activated by being selected and is not affected.